src/main/java/com/limechain/network/protocol/warp/dto/BlockHeader.java

```diff
@@ -1,21 +1,22 @@
 package com.limechain.network.protocol.warp.dto;
 
 import com.limechain.network.protocol.blockannounce.scale.BlockHeaderScaleWriter;
-import com.limechain.utils.scale.exceptions.ScaleEncodingException;
 import com.limechain.utils.HashUtils;
-import io.emeraldpay.polkaj.scale.ScaleCodecWriter;
+import com.limechain.utils.scale.ScaleUtils;
 import io.emeraldpay.polkaj.types.Hash256;
 import lombok.Getter;
 import lombok.Setter;
 
-import java.io.ByteArrayOutputStream;
-import java.io.IOException;
 import java.math.BigInteger;
 import java.util.Arrays;
 
 @Setter
 @Getter
 public class BlockHeader {
+    // TODO: Make this const configurable
+    public static final int BLOCK_NUMBER_SIZE = 4;
```

Comment:
I'm pretty sure this isn't where we want this const to reside, but temporarily, I found this the most sensible place.
See [the issue regarding this](https://github.com/orgs/LimeChain/projects/16?pane=issue&itemId=53479974).